packages/flutter/lib/src/widgets/image.dart

```diff
@@ -1075,28 +1073,28 @@ class Image extends StatefulWidget {
 }
 
 class _ImageState extends State<Image> with WidgetsBindingObserver {
-  ImageStream _imageStream;
-  ImageInfo _imageInfo;
-  ImageChunkEvent _loadingProgress;
+  ImageStream? _imageStream;
+  ImageInfo? _imageInfo;
+  ImageChunkEvent? _loadingProgress;
   bool _isListeningToStream = false;
-  bool _invertColors;
-  int _frameNumber;
-  bool _wasSynchronouslyLoaded;
-  DisposableBuildContext<State<Image>> _scrollAwareContext;
-  Object _lastException;
-  StackTrace _lastStack;
+  late bool _invertColors;
+  int? _frameNumber;
+  bool? _wasSynchronouslyLoaded;
```

Comment:
looks like this one should be non-nullable, maybe just default to false?

Reply:
Done